### PR TITLE
Bottom Footer CTA with gradient

### PR DESCRIPTION
**Pull request by alanlanglois**
## Detailed purpose of the PR
Bottom footer CTA with gradient. Let you scroll the view behind AND click on the button. 

### Result and observation
button-bar

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra dependency ?**
      If checked, Which extra dependency did you add ? (name, purpose, link ...).

### Testing Strategy

- [ ] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: src/components/button-footer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// @flow strict
+
+import * as React from 'react';
+import {View, StyleSheet} from 'react-native';
+
+import type {Layout} from '../containers/with-layout';
+import theme from '../modules/theme';
+// import withLayout from '../containers/with-layout';
```
Comment (VincentCATILLON) [inline src/components/button-footer.js]: should be removed

File: src/components/button-footer.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    width: '100%'
+  },
+  cta: {
+    position: 'absolute',
+    bottom: theme.spacing.base,
+    left: theme.spacing.base
+  }
+});
+
+const ButtonFooter = ({children, onPress, testID, layout, onLayout}: Props) => {
```
Comment (VincentCATILLON) [inline src/components/button-footer.js]: COMMENT: IMO, ButtonSticky is clearer

File: src/components/level-end.js [hunk 3]
```diff
@@ -17,7 +19,7 @@ import CatalogItem from './catalog-item';
 import Starburst from './starburst';
 import HeartBroken from './heart-broken';
 import Trophy from './trophy';
-import Button from './button';
+import {HEIGHT as ButtonHeight} from './button';
```
Comment (VincentCATILLON) [inline src/components/level-end.js]: constant are in uppercase please

File: src/components/level-end.js [hunk 4]
```diff
@@ -147,74 +150,83 @@ class LevelEnd extends React.PureComponent<Props> {
...
-                  {translations.outOfLives}
-                </Text>
-              )}
-            </View>
-            {isSuccess ? (
-              <Trophy style={[styles.icon, {height: screenWidth}]} />
-            ) : (
-              <HeartBroken style={[styles.icon, {height: screenWidth}]} />
-            )}
+          <View style={{flex: 1}}>
```
Comment (VincentCATILLON) [inline src/components/level-end.js]: Bad style

File: src/components/button-footer.js (hunk 2 above)
Comment (alanlanglois) [reply src/components/button-footer.js]: @VincentCATILLON should I rename the whole component then?

**Pull request merged**